packages/ui-react/src/pages/User/User.tsx

```diff
@@ -162,8 +182,11 @@ const User = (): JSX.Element => {
               }
             />
           )}
-          <Route path="payments" element={accessModel !== ACCESS_MODEL.AVOD ? <PaymentContainer /> : <Navigate to="my-account" />} />
-          <Route path="*" element={<Navigate to="my-account" />} />
+          <Route
+            path="payments"
```

Comment:
Missing nested route (favorites on L156 as well)? 😄

Reply:
Sharp! Thank you.